leases/schema.py

```diff
@@ -65,12 +72,10 @@ class Input:
     berth_lease = graphene.Field(BerthLeaseNode)
 
     @classmethod
-    @login_required
-    @superuser_required
+    @view_permission_required(BerthApplication, Boat)
```

Comment:
replace `Boat` with `CustomerProfile`, since you can access the entire profile from the lease